### ISSUE TITLE
All code across all slides should pass rubocop.

### ISSUE THREAD

**Issue by vinyar** [chef-training/chef-fundamentals-linux#8]
presently the code in the slides will not pass rubocop and some of the code uses old ruby syntax.

There are many examples where double `"` quotes are used instead of single `'` .
Handouts use tabs instead of double spaces.
etc...

**Comment by eeyun** [comment]
I agree with this personally. Theres no reason that we should be teaching people bad practices and explaining it away with hand-waving. +1 @vinyar . I'm submitting a new issue on the chef-fundamentals-online repo as well.
